plasmoid/contents/ui/main.qml

```diff
@@ -221,6 +216,30 @@ Item {
                         value: brightness
                         stepSize: plasmoid.configuration.stepSize || 1
 
+                        onValueChanged: function() {
+                            if (brightness != value) {
+
+                                // Round value in step
+                                brightness = roundNumberInStep(value, stepSize)
```

Comment:
The reason this is needed is that if a user manually adjusts the brightness by display's panel, it may not match the `stepSize`.

`onWheel` will increase/decrease value by `stepSize`. For example, if the current `value` is 1, then adding `stepSize = 10` will make it 11. `roundNumberInStep` will round 11 to 10.